pkg/config/config_test.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadConfig(t *testing.T) {
+	var tmpDir string
+
+	setup := func() {
+		tmpDir = t.TempDir()
+	}
+
+	teardown := func() {
+		os.RemoveAll(tmpDir)
+	}
```

Comment:
I'd recommend move all this down into individual test cases. In the future, we'll likely want to add `t.Parallel` to run parallel tests. And having any shared resource among tests (even a sole var like this) could cause problems.

Generally, I like to keep tests as flake free as possible with much their own resources.